.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
```

Comment:
Circle-CI translates this config into a different format: resolves the &defaults reference and apparently inlines the "install" command. Is there an expectation that "defaults" or "install" objects will be reused to justify breaking them out?

 https://circleci.com/gh/openworm/openworm_docs/4#config/containers/0
Is there

Reply:
not for this PR, but potentially for addressing #55. Could remove this for now if you prefer.